utils/graphql/schema.ts

```diff
@@ -0,0 +1,289 @@
+        return resolver.recordById(args.id);
+      },
+    },
+    records: {
+      type: GraphQLList(type.RecordType),
+      description: "List of All Records",
+      resolve: () => {
+        return resolver.allRecords();
+      },
+    },
+    transactionById: {
+      type: type.TransactionType,
+      description: "A Single Transaction by its ID",
+      args: {
+        id: { type: GraphQLInt },
+      },
+      resolve: (_, args) => {
+        return resolver.transactionById(args.id);
+      },
+    },
+    transactions: {
+      type: GraphQLList(type.TransactionType),
+      description: "List of All Transactions",
+      resolve: () => {
+        return resolver.allTransactions();
+      },
+    },
+    batchById: {
+      type: type.BatchType,
+      description: "A Single Batch by its ID",
+      args: {
+        id: { type: GraphQLInt },
+      },
+      resolve: (_, args) => {
+        return resolver.batchById(args.id);
+      },
+    },
+    batches: {
+      type: GraphQLList(type.BatchType),
+      description: "List of All Batches",
+      resolve: () => {
+        return resolver.allBatches();
+      },
+    },
+    accountById: {
+      type: type.AccountType,
+      description: "A Single Account by its ID",
+      args: {
+        id: { type: GraphQLInt },
+      },
+      resolve: (_, args) => {
+        return resolver.accountById(args.id);
+      },
+    },
+    accounts: {
+      type: GraphQLList(type.AccountType),
+      description: "List of All Accounts",
+      resolve: () => {
+        return resolver.allAccounts();
+      },
+    },
+    bankPageRecordById: {
+      type: type.BankPageRecordType,
+      description: "A Single Bank Page Record by its ID",
+      args: {
+        id: { type: GraphQLInt },
+      },
+      resolve: (_, args) => {
+        return resolver.bankPageRecordById(args.id);
+      },
+    },
+    bankPageRecords: {
+      type: GraphQLList(type.BankPageRecordType),
+      description: "List of All Bank Page Records",
+      resolve: () => {
+        return resolver.allBankPageRecords();
+      },
+    },
+    bankPageById: {
+      type: type.BankPageType,
+      description:
+        "A Single Bank Page (Which Is A List Of Bank Page Records), by its ID",
+      args: {
+        id: { type: GraphQLInt },
+      },
+      resolve: (_, args) => {
+        return resolver.bankPageById(args.id);
+      },
+    },
+    bankPages: {
+      type: GraphQLList(type.BankPageType),
+      description: "List Of Bank Pages",
+      resolve: () => {
+        return resolver.allBankPages();
+      },
+    },
+  }),
+});
+
+const RootMutationType = new GraphQLObjectType({
+  name: "Mutation",
+  description: "Root Mutation",
+  fields: () => ({
+    userCompanies: {
+      type: GraphQLList(type.CompanyType),
+      description:
+        "List of Companies for user token thats defined on: 'WizcloudApiPrivateKey'",
+      resolve: () => {
+        return hashavshevet.getCompanies();
```

Comment:
why is getting companies a mutation and not a query?

Reply:
there's actually no reason.
I automatically placed all of Hashavshevet organic API calls as mutations because its all POST calls. but you are right.